haystack/utils/deepsetcloud.py

```diff
@@ -85,6 +86,7 @@ def post(
         self,
         url: str,
         json: dict = {},
+        data: Any = None,
```

Comment:
Out of curiosity, why did you type it explicitly as `Any` all around? I thought mypy would assume `Any` if no type was specified

Reply:
So it doesn't look like someone simply forgot to add an appropriate typing.